django/db/migrations/autodetector.py

```diff
@@ -1038,6 +1088,7 @@ def _generate_altered_foo_together(self, operation):
                 )
 
     def generate_altered_unique_together(self):
+        # TODO: Add deprecation warnings here.
```

Comment:
Does this method need to go through a deprecation cycle? I am not sure if it's consider as a part of the public API.